_html/scss/modules/_error-content.scss

```diff
@@ -0,0 +1,45 @@
+.error-page-container {
+    background-color: #2b101c;
+    height: 100vh;
+}
+
+.image-text-wrapper {
+    text-align: center;
+    width: 70%;
+
+    @include position (absolute, 25%, auto, auto, auto);
```

Comment:
Since horizontal-center mixin already has position absolute, you can just replace this line with `top: 25%`